data-access-api/src/gov/nasa/jpl/labcas/data_access_api/service/DownloadServiceImpl.java

```diff
@@ -84,6 +84,10 @@ public Response download(@Context HttpServletRequest httpRequest, @Context Conta
 				LOG.info(doc.toString());
 				fileLocation = (String)doc.getFieldValue(SOLR_FIELD_FILE_LOCATION);
 				fileName = (String)doc.getFieldValue(SOLR_FIELD_FILE_NAME);
+				name=(String)doc.getFieldValue(SOLR_FIELD_NAME);
```

Comment:
Stylistically I like to see spaces around `=` when used in assignment operators, comparisons, etc.—but you're lucky we don't have a styleguide! 😝

Reply:
:P